src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -119,6 +119,12 @@ public void startTask(final String tableName, final TopicPartition topicPartitio
     }
   }
 
+  @Override
+  public void startPartitions(
+      Collection<TopicPartition> partitions, Map<String, String> topic2Table) {
+    partitions.forEach(tp -> this.startTask(Utils.tableName(tp.topic(), topic2Table), tp));
```

Comment:
we should rename this to `startPartition` instead of `startTask`

Let's create a jira and add a TODO here. You can do this if you have time left after your current PRs are merged.

Reply:
Renamed